admin-dev/themes/default/css/overrides.css

```diff
@@ -1,2 +1,2 @@
 /* This stylesheet should be used to add your custom styles to the back-office without using the Sass sources. It will be loaded after all the default styles. 
-You should NOT edit any other exisiting back-office CSS file manually: they are generated by the Sass preprocessor: http://www.sass-lang.com/ . */
\ No newline at end of file
+You should NOT edit any other exisiting back-office CSS file manually: they are generated by the Sass preprocessor: http://www.sass-lang.com/ . */
```

Comment:
I accidentally added this file. I don't want to change and I am not sure what's the change is to remove it.

Reply:
It seems the only change you did was adding an empty blank line at the end, which complies with the standard for this.

Reply:
I am trying to fix this in GitHub UI but it fails

Reply:
If I try to copy the whole previous file, PHPStorm just "fixes" it for me automatically I tried to remove the blank line via nano but it didn't help either. It's annoying little thing to tackle

Reply:
@JevgenijVisockij Fixed.
I checked out your branch, opened the file with Sublime which is the most awesome text editor ever 😄  removed the empty line and saved.